pennylane/debugging.py

```diff
@@ -53,18 +62,23 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         self.device._debugger = None
 
 
-def snapshots(qnode):
-    r"""Create a function that retrieves snapshot results from a QNode.
+@transform
+def snapshots(
+    tape: QuantumTape, shots: Optional[int] = None
```

Comment:
Does this `shots` keyword argument still do anything?  I'm thinking it can be removed.

Reply:
Yeah you are right. It was overshadowed by the local variable and the linter missed it 👍